src/chains/supportedChains.evm.ts

```diff
@@ -1160,7 +1160,7 @@ export const supportedEVMChains: EVMChain[] = [
     metamask: {
       chainId: prefixChainId(80094),
       blockExplorerUrls: ['https://berascan.com/', 'https://beratrail.io/'],
-      chainName: 'Berachain Mainnet',
+      chainName: 'Berachain',
```

Comment:
Thanks for your help with the new chain, but I have to revert the name, since these information is used to add new chain to our customers, we'd better follow the data provide by the https://berascan.com/
<img width="354" alt="image" src="https://github.com/user-attachments/assets/416eca0b-d180-4c83-8a29-3dcfeb18fbcf" />

Reply:
just ignore the above lol